template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/deployment/batch_inference/predict.py.tmpl

```diff
@@ -11,21 +11,22 @@ def predict_batch(
     """
     {{ if (eq .input_include_models_in_unity_catalog "yes") }}mlflow.set_registry_uri("databricks-uc"){{ end }}
     table = spark_session.table(input_table_name)
-    {{ if (eq .input_include_feature_store `yes`) }}
-    from databricks.feature_store import FeatureStoreClient
-    
+    {{ if (eq .input_include_feature_store `yes`) }} 
```

Comment:
nit space

Reply:
see latest updates -- will be adding links to screenshots